src/blocks/FilterBlock/__stories__/data.json

```diff
@@ -0,0 +1,29 @@
+{
+  "default": {
+    "card": {
+      "type": "card-with-image",
```

Comment:
Maybe filter-block?

Reply:
No, this is a template used to generate cards for inner block.